addons/web_editor/static/tests/html_field_tests.js

```diff
@@ -797,4 +797,62 @@ QUnit.module("WebEditor.HtmlField", ({ beforeEach }) => {
         assert.strictEqual(p.innerText.replaceAll('\u200B', ''), 'New label',
             "The link's label should be updated");
     });
+
+    QUnit.module("Dropdown");
```

Comment:
Are there no colorpicker tests yet where this could have been added instead?
cc @dmo-odoo

Reply:
I searched in the code and I didn't find any